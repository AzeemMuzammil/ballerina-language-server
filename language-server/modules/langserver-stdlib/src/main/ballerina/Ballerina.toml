[package]
org = "ballerina"
name = "langserverstdlib"
version = "0.1.0"
