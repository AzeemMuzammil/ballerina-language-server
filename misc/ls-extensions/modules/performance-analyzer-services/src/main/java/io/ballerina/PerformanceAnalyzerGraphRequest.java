package io.ballerina;

import org.eclipse.lsp4j.Range;
import org.eclipse.lsp4j.TextDocumentIdentifier;

public class PerformanceAnalyzerGraphRequest {
    private TextDocumentIdentifier documentIdentifier;
    private Range range;
    private String choreoToken;
    private String choreoCookie;

    public TextDocumentIdentifier getDocumentIdentifier() {

        return documentIdentifier;
    }

    public void setDocumentIdentifier(TextDocumentIdentifier documentIdentifier) {

        this.documentIdentifier = documentIdentifier;
    }

    public Range getRange() {

        return range;
    }

    public void setRange(Range range) {

        this.range = range;
    }

    public String getChoreoToken() {

        return choreoToken;
    }

    public void setChoreoToken(String choreoToken) {

        this.choreoToken = choreoToken;
    }

    public String getChoreoCookie() {

        return choreoCookie;
    }

    public void setChoreoCookie(String choreoCookie) {

        this.choreoCookie = choreoCookie;
    }
}
