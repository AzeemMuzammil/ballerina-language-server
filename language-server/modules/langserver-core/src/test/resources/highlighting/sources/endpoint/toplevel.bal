import ballerina/http;

http:Client myClient = new("");
