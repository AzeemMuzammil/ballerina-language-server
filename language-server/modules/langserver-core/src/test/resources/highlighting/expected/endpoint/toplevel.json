{
  "source": {
    "file": "toplevel.bal"
  },
  "result": [
    {
      "line":2,
      "token": "WzEyLCA4LCAwXQ=="
    }
  ]
}